src/main/java/com/gary/backendv2/controller/IncidentController.java

```diff
@@ -42,4 +43,10 @@ public ResponseEntity<?> addAmbulances(@PathVariable Integer id, @RequestBody Li
 		incidentService.addAmbulances(id, licencePlates);
 		return ResponseEntity.ok("Ambulances assigned successfully to incident");
 	}
+
+	@PutMapping("/{id}/status")
+	public ResponseEntity<?> changeStatus(@PathVariable Integer id, @RequestBody IncidentStatusType incidentStatusType){
+		incidentService.changeIncidentStatus(id, incidentStatusType);
+		return ResponseEntity.ok("Status successfully changed");
+	}
 }
```

Comment:
Inny path bym tutaj zdefiniował. 
`/{id}/status/{status}`
ładniejsze sematycznie, bo nie trzeba przesyłać requestu z jednym polem w body i bardziej czytelne bo samego URL było by widać na jaki status zmieniamy

oraz PUT na POST

The difference between PUT and POST is that PUT is idempotent: **calling it once or several times successively has the same effect** (that is no side effect),

u nas wysłanie 2 razy tego samego rządania z tym samym stanem za pierwszym razem by przeszło, a za drugim rzuci http 403